src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCar = 40;
+  let discount = 0;
```

Comment:
Since 'discount' is not reassigned, you can use 'const' instead of 'let' to prevent unintentional changes, as per the checklist item: 'Prefer const over let where possible, to avoid unintentional changes.'